packages/browser/README.md

```diff
@@ -1,97 +1,11 @@
-You'd use the single session API to:
-
-- Create a single page web application that only one user logs into at a time.
-- Make a bot that only represents one user.
-- Build a mobile application that only one user logs into at a time.
-
-#### Examples
-
-- [Single-session the web browser with your own bundler](examples/single/bundle)
-
-### Multi Session API
-
-`solid-client-authn`'s "Multi Session API" allows you to manage multiple sessions for multiple logged in users at the same time. You'd use the multi session API to:
-
-- Create a web application with a server side component. While one user may be logged into each individual client, the server will need to manage all logged in users.
-- Make a bot that needs to operation on behalf of multiple users.
-- You have a single page application, but you want to maintain credentials on the server for security.
-- You want to log in multiple identities for either a single human user, or multiple users.
-
-### Setting up the examples
-
-```bash
-git clone https://github.com/inrupt/solid-client-authn.git
-cd solid-client-authn
-npm ci
-npm run bootstrap-examples
-# Run each example from the root
-## single session API, bundler mode
-cd examples/single/bundle
-npm run start
-## single session API, script mode
-cd examples/single/script
-npm run start
-```
-
-At this point, a test application will be running on port `3001`.
-Be sure that you enter a valid Solid issuer before logging in
-, for example: `https://identity.demo-ess.inrupt.com`.
-
-# Issues & Help
-
-## Solid Community Forum
-
-If you have questions about working with Solid or just want to share what you’re working on, visit the [Solid forum](https://forum.solidproject.org/). The Solid forum is a good place to meet the rest of the community.
-
-## Bugs and Feature Requests
-
-- For public feedback, bug reports, and feature requests please file an issue via [GitHub](https://github.com/inrupt/solid-client-authn/issues/).
-- For non-public feedback or support inquiries please use the [Inrupt Service Desk](https://inrupt.atlassian.net/servicedesk).
-
-## Prerequisite
-
-Any of the `solid-client-authn` libraries require at least:
-
-- NodeJS 12.X.Y
-- npm 6.14.X
-  **Note**: We recommand using [nvm](https://github.com/nvm-sh/nvm) to manage your node version
-
-The `solid-client-authn` libraries are compatible with [NSS](https://github.com/solid/node-solid-server/releases/tag/v5.3.0) 5.3.X and higher.
-
-## Documentation
-
-- [Using the libraries from within the browser](./docs/browser.md)
-- [Inrupt documentation Homepage](https://docs.inrupt.com/)
+`solid-client-authn-browser` is based on [`oidc-client-js`](https://github.com/IdentityModel/oidc-client-js). 
+However, the latter lacks some features that are necessary to provide the developer
+experience we specifically want for the Solid ecosystem, so we developped  
+[`oidc-dpop-client-browser`](../oidc-dpop-client-browser/README.md) to add these features.
```

Comment:
The README is also visible on npmjs.com, for example, where a relative link won't work.

```suggestion
[`oidc-dpop-client-browser`](https://www.npmjs.com/package/@inrupt/oidc-dpop-client-browser) to add these features.
```